### DISCUSSION TITLE
Display lfView on a sphere for 360 - Vision Pro

### DISCUSSION THREAD

**Discussion by LudoTexx** [HaishinKit/HaishinKit.swift#1520] [Q&A]
Hello,

Is this possible to display lfView into a sphere for displaying a 360 (or 180) video inside Apple Vision Pro?

I'm a beginner in swift and was not able to get the material from lfView for using it inside ModelEntity in my RealityView.

Thank in advance.

**Comment by shogo4405** [comment]
I'm not sure. I’m not familiar with the technology related to Vision Pro.
